src/components/issue-page/index.tsx

```diff
@@ -13,27 +12,41 @@ export type LoadedIssueContentProps = {
 }
 
 function useDemo(issue: Issue, postComment: LoadedIssueContentProps['postComment'], changeStatus: LoadedIssueContentProps['changeStatus']): void {
-  if (issue.status === 'opened') {
-    setTimeout(() => {
-      changeStatus({ username: 'devdiva', avatarUrl: '/devdiva.png' }, 'acknowledged', {
-        html: `<div id="diva-acknowledged">I am able to reproduce this on our end, sorry about that! We'll get working on a fix right away</div>`,
-      })
-    }, 3000)
-  }
-
   if (issue.status === 'acknowledged') {
     setTimeout(() => {
       changeStatus({ username: 'devdiva', avatarUrl: '/devdiva.png' }, 'closed', {
         html: `<div>Deployed a fix that seems to be fixing the issue. Definitely leave a comment and reopen this issue if you're still being affected!</div>`,
       })
-    }, 6000)
+    }, 3000)
   }
 }
 
 export default function LoadedIssueContent({ avatarUrl, issue, postComment, changeStatus }: LoadedIssueContentProps): JSX.Element {
   useDemo(issue, postComment, changeStatus)
 
-  const tour = useTour(tourArgs)
+  const steps = Object.freeze([
+    {
+      text: ['The issue is now posted to the more human internet platform where the site’s maintainer can see and address it.'],
+      attachTo: {
+        element: '.issue-timeline',
+        on: 'top',
+      },
+      ...withNextButton,
```

Comment:
Let's do the same thing for devdiva closing the issue and eliminate the `useDemo` effect entirely.